controllers/assign_book.go

```diff
@@ -0,0 +1,37 @@
+package controllers
+
+import (
+	"l-m-s/models"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+)
+
+// This function handles http request to assign a book to a student.
+func AssignBook(c *gin.Context) {
+
+	stuId := c.PostForm("student_id")
+	bookId := c.PostForm("book_id")
+
+	studentID, err := strconv.Atoi(stuId)
```

Comment:
what if user gives student_id  = 0 and book_id  = 0 or negative numbers ?
add validations.